sheet.go

```diff
@@ -1753,6 +1752,14 @@ func (f *File) GetPageLayout(sheet string) (PageLayoutOptions, error) {
 //	    RefersTo: "Sheet1!$1:$1",
 //	    Scope:    "Sheet1",
 //	})
+//
+// You can also use the function in RefersTo. For example:
+//
+//	err := f.SetDefinedName(&excelize.DefinedName{
```

Comment:
Remove double quote after plus sign, and using new Name different with other examples would be better. I suggest adding example like this:

```text
err := f.SetDefinedName(&excelize.DefinedName{
    Name:     "CustomRange",
    RefersTo: "Sheet1!$A$2+Sheet1!$D$5",
    Scope:    "Sheet1",
})
```